Header header

string ok_agent_identifier  		# for receive_ok method - empty
customPoint ok_value  			# for receive_ok method -1,1,1
agentViewElement[] no_goods  		# for receieve_no_good method

bool ok #false
bool nogood #false

